src/variables.scss

```diff
@@ -26,3 +26,15 @@ $danger-dark: var(--qr-color-danger-dark, hsl(0, 72%, 51%));
   font-size: 20px;
   line-height: 28px;
 }
+
+@mixin screen-size-s-and-up($width: 580px) {
+  @media (min-width: $width) {
+    @content;
+  }
+}
+
+@mixin screen-size-sm-and-up($width: 800px) {
```

Comment:
Did you take the naming from the web project? Here it probably makes more sense to go with `screen-size-s-and-up` and `screen-size-m-and-up` because there is no in-between.

Out of curiosity, where do the values `580px` and `800px` come from?

Reply:
Both the values and the naming come from web (these are the values the WLDX agency design suggested).
These widths are quite close to each other but I guess we can rename it again if we need to introduce higher values in the future, which I actually doubt... I'll change this to `m`.